src/components/ui/Collapsible.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, ReactElement } from "react";
+import ArrowRight from "../../assets/icons/arrow-right.svg";
+import ArrowDown from "../../assets/icons/arrow-down.svg";
+
+type CollapsibleProps = React.PropsWithChildren<{
+  height?: number;
+  width?: number;
+  children: ReactElement;
+}>;
+
+const Collapsible = ({
+  height,
+  width,
+  children,
+}: CollapsibleProps): ReactElement => {
+  const Open = () => <ArrowDown width={height} height={width}></ArrowDown>;
+  const Closed = () => <ArrowRight width={height} height={width}></ArrowRight>;
+  const [open, setOpen] = useState(false);
+  const [icon, setIcon] = useState(<Closed />);
+  const toggle = () => {
+    const newIcon = !open ? <Open /> : <Closed />;
+    setIcon(newIcon);
```

Comment:
Micro nit: less variables is better, since that's less stuff to keep in my head when reading the code. Not something that's without contention, feel free to defer.

```suggestion
    setIcon(!open ? <Open /> : <Closed />);
```